velox/exec/PartitionedOutput.cpp

```diff
@@ -184,7 +191,10 @@ void PartitionedOutput::initializeDestinations() {
     auto taskId = operatorCtx_->taskId();
     for (int i = 0; i < numDestinations_; ++i) {
       destinations_.push_back(std::make_unique<detail::Destination>(
-          taskId, i, pool(), eagerFlush_));
+          taskId, i, pool(), eagerFlush_, [&](uint64_t bytes, uint64_t rows) {
+            auto lockedStats = stats_.wlock();
+            lockedStats->addOutputVector(bytes, rows);
+          }));
```

Comment:
Is this ok to capture this in a lambda?
Could that lead to crashes when this gets destroyed and the callback called?

Reply:
Should be Ok since all operations are synchronous.